<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDoctorForm</class>
 <widget class="QWidget" name="AddDoctorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>135</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="de_doctor_birthdate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>55</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_doctor_sex">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>165</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>f</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>175</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_doctor_lastname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_doctor_firstname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>95</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doctor_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>255</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>215</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Specialisation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_doctor_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>255</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_doctor_specialisation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
